stories/dynamic-page/dynamic-page.stories.js

```diff
@@ -0,0 +1,929 @@
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+            </div>
+        </div>
+        <div class="fd-bar fd-bar--floating-footer">
+            <div class="fd-bar__right">
+                <div class="fd-bar__element">
+                    <button aria-label="button" class="fd-button fd-button--emphasized fd-button--compact">Save</button>
+                </div>
+                <div class="fd-bar__element">
+                    <button aria-label="button" class="fd-button fd-button--transparent fd-button--compact">Cancel</button>
+                </div>
+            </div>
+        </div>
+    </div>
+    <br /><br />
+`;
+
+tablet.storyName = 'Tablet mode';
+tablet.parameters = {
+    docs: {
+        iframeHeight: 150,
+        storyDescription: 'The dynamic page with Tablet specification(L). Use the `--m` modifier on `title-area`, `breadcrumb`, ' +
```

Comment:
The dynamic page displays on tablet screens with the L size. To optimize dynamic page for tablet, add these modifier classes to the following elements:

Element | Modifier class
:-- | :--
Title Area | `fd-dynamic-page__title-are--m`
Header | `fd-dynamic-page__header--m`
Content | `fd-dynamic-page__content--m`

It is recommended that the navigation actions are displayed in a separate row above global actions, and are placed at the same level as the breadcrumb component.

Reply:
I didn't see the modifier class attached to the Breadcrumb, maybe I missed it?

Reply:
Since breadcrumb is part of title area, applying modifier to title area is sufficient; breadcrumbs don't need these modifiers. However, there is a specific padding applied for `title` element, and have added a row for this additionally.